### DISCUSSION TITLE
Mod to remove long notes on Mania

### DISCUSSION THREAD

**Discussion by KGPatt** [ppy/osu#15299] [Ideas] [answered]
Could there possibly be the addition of a long note removal mod? In the early access VSRG, Quaver, there is a mod that replaces the beginning and end notes of a long note with a normal note, and I feel like this could be a good addition to Mania on Lazer.

**Comment by EVAST9919** [comment, marked as answer]
There's a `Hold Off` mod in mania which does exactly that. Not sure if it was added before or after this thread was created though.

**Reply by mk56-spn** [reply]
doesnt matter much but i checked out of curiousity and this discussion is 4 months older, so the suggestion here is understandable
